web/src/main/webapp/WEB-INF/classes/log4j2-index.xml

```diff
@@ -43,7 +55,9 @@
     <Logger name="geonetwork.geoserver.publisher" level="error"/>
     <Logger name="geonetwork.geoserver.rest" level="error"/>
     <Logger name="geonetwork.harvest.wfs.features"/>
-    <Logger name="geonetwork.harvester" level="info"/>
+    <Logger name="geonetwork.harvester" level="info">
+      <AppenderRef ref="Harvester"/>
+    </Logger>
```

Comment:
So you are only logging harvester things? I thought the previous approach harvested "all things" while the harvest context was active?

Reply:
I expected the harvester file to pick up all log messages (not only those set to harvester execution). But it appears that expectation was based on https://github.com/geonetwork/core-geonetwork/pull/6523